models/dtl/wrt/census_JSON_store.py

```diff
@@ -0,0 +1,35 @@
+from dtl.api.dtl_census import census_get
+from est.db.cur import con_cur
+from dtl.wrt.jprint import jprint
+from io import StringIO
+import json
+import psycopg2
+from psycopg2 import sql
+
+###Not yet working; figuring out json data input with pop_table first
```

Comment:
The industry standard for these styles of comments is `TODO: <blah-blah-blah>`. There is tooling around this 😄

Reply:
So you're saying ... it helps to be organized and not leave random shit everywhere? woops